scripts/build.js

```diff
@@ -0,0 +1,70 @@
+const childProcess = require("child_process");
+const path = require("path");
+const { promisify } = require("util");
+
+const exec = promisify(childProcess.exec);
+
+async function run(argv) {
+  const { bundle = "stable", outDir: relativeOutDir = "./lib", verbose } = argv;
+
+  const env = {
+    NODE_ENV: "production",
+    BABEL_ENV: bundle,
+  };
+
+  const babelConfigPath = path.resolve(__dirname, "../babel.config.js");
+  const srcDir = path.resolve("./src");
+
+  const extensions = [".js", ".ts", ".tsx"];
+  const ignore = [
+    "**/*.test.js",
+    "**/*.test.ts",
+    "**/*.test.tsx",
+    "**/*.spec.ts",
+    "**/*.spec.tsx",
+    "**/*.d.ts",
+  ];
+
+  const outDir = path.resolve(
+    relativeOutDir,
+    {
+      node: "./",
+      stable: "./esm",
+    }[bundle]
+  );
+
+  const babelArgs = [
+    "--config-file",
+    babelConfigPath,
+    "--extensions",
+    `"${extensions.join(",")}"`,
+    srcDir,
+    "--out-dir",
+    outDir,
+    "--ignore",
+    `"${ignore.join('","')}"`,
+  ];
+
+  const command = ["cross-env babel", ...babelArgs].join(" ");
```

Comment:
I don't think we need `cross-env` here unless we need to add `cross-env NODE_ENV=production` in (Which I don't think we do as it's being set below)

Reply:
I actually got some errors when I simply wanted to run `npm run babel` thats why I decided to stick with `cross-env`